src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -210,6 +215,7 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
       startBlock,
       batch
     )
+    let wasBatchTruncated = false
```

Comment:
Add comment: //This is to prevent against a case where a batch is oversized, but then gets truncated to the point where it is under the minimum size. In this case, we want to submit regardless